freshmeat/freshmeat.py

```diff
@@ -57,7 +57,7 @@ async def freshmeat(self, ctx, hours: int = 24):
             embed = discord.Embed(description=page)
             embed.set_author(
                 name=f"{ctx.author.display_name}'s freshmeat of the day.",
-                icon_url=ctx.author.avatar_url_as(format="png")
+                icon_url=ctx.author.avatar.url
```

Comment:
`avatar` can be `None`, we want `display_avatar` here:
```suggestion
                icon_url=ctx.author.display_avatar,
```